src/ln/peer_handler.rs

```diff
@@ -1180,4 +1271,67 @@ mod tests {
 		peers[0].process_events();
 		assert_eq!(peers[0].peers.lock().unwrap().peers.len(), 0);
 	}
+
+ 	#[test]
+	fn test_ping_peers(){
```

Comment:
For the tests, if would try to spawn 2 nodes, connect them, call check_peer as much as needed on PeerHandler to schedule a ping, than call check_peer a number of times equal to WAIT_PONG_UNTIL_BEFORE_DISCONNECT to see if peer has well replied by checking it hasn't been disconnected.